src/components/TodoFooter/TodoFooter.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import { Filter } from '../../types/Filter';
+
+type Props = {
+  todos: Todo[];
+  filter: Filter;
+  onFilterChange: (filter: Filter) => void;
+  onDeletedCompleted: () => void;
+};
+
+const FILTER_LINKS: {
+  label: string;
+  value: Filter;
+  href: string;
+  dataCy: string;
+}[] = [
+  { label: 'All', value: Filter.All, href: '#/', dataCy: 'FilterLinkAll' },
+  {
+    label: 'Active',
+    value: Filter.Active,
+    href: '#/active',
+    dataCy: 'FilterLinkActive',
+  },
+  {
+    label: 'Completed',
+    value: Filter.Completed,
+    href: '#/completed',
+    dataCy: 'FilterLinkCompleted',
+  },
```

Comment:
Why do you create additional array? You can use `Object.values(Filter)` for creating arrauy